geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/ConverterConfigResolver.scala

```diff
@@ -10,61 +10,72 @@ package org.locationtech.geomesa.convert
 
 import java.io.File
 
-import com.typesafe.config.{Config, ConfigFactory, ConfigParseOptions}
+import com.typesafe.config.{Config, ConfigFactory}
 import com.typesafe.scalalogging.LazyLogging
+import org.locationtech.geomesa.utils.conf.AbstractArgResolver
 
 import scala.util.{Failure, Success, Try}
 
 /**
  * Attempts to resolve Converter config from arguments as either a string or
  * as a filename containing the converter config
  */
-object ConverterConfigResolver extends LazyLogging {
+case class ConfArgs(config: String)
+object ConverterConfigResolver extends AbstractArgResolver[Config, ConfArgs] with LazyLogging {
```

Comment:
I think we do for the abstraction to work. SftArgResolver has 2 params in it's case class.